### DISCUSSION TITLE
file uploaded from minio UI changed to directory

### DISCUSSION THREAD

**Discussion by pravesh-mathema** [minio/minio#18152] [Q&A] [answered]
Hello,

I have installed latest minio server and created one bucket. 
I have uploaded  png file from Minio UI to created bucket but that file changed to directory.

So requesting for the help.

**Comment by shtripat** [comment]
Can you please elaborate the issue more. Attach some screenshots etc if available. Are you able to download the object from console or using MinIO client `mc`?

**Comment by ravindk89** [comment, marked as answer]
This is an expected behavior resulting from the Erasure Coding system.

As per our [Documentation](https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html), we have deprecated the "Filesystem" backend where files were kept as-is on single drive deployments.

For single drive, we now use an `EC:0` zero parity erasure coded backend. The files are erasure coded - if you `ls` the directory you should see an `xl.meta` and a file UUID. That is the data.
